core-bundle/src/Messenger/WebWorker.php

```diff
@@ -141,4 +148,16 @@ private function processTransport(string $transportName): void
 
         $this->webWorkerRunning = false;
     }
+
+    private function getRemainingExecutionTime(): int
+    {
+        $maxTime = (int) ini_get('max_execution_time');
+
+        if (1 > $maxTime) {
+            return PHP_INT_MAX;
+        }
+
+        // Substract already used up execution time
+        return $maxTime - (microtime(true) - ($_SERVER['REQUEST_TIME_FLOAT'] ?? microtime(true)));
```

Comment:
We should probably use something different than `$_SERVER['REQUEST_TIME_FLOAT']` here?

Reply:
I wouldn't know what 🤔

Reply:
Ah yes that would be the proper accessor but still there might be many SAPIs or setups that do not support that CGI variable. But then it would just be `null` and that's covered so 👍

Reply:
Looks like Symfony provides a default for us 🎉 https://github.com/symfony/symfony/blob/9ad708b6963a42a6a7172e9c1d5cfbe847051756/src/Symfony/Component/HttpFoundation/Request.php#L295